### PR TITLE
yq: Use install_yq.sh script from tests repository

### PR DESCRIPTION
**Pull request by chavafg**
Removes two (similar) functions that install `yq`. Instead of
having different functions, use the one that we have in the
tests repository.
In addition, removes the `.ci/lib.sh` which only had an additional
`clone_tests_repo` function which was not being used.

Fixes: #939.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

File: scripts/lib.sh [hunk 1]
```diff
@@ -1,33 +1,31 @@
...
 hub_bin="hub-bin"
 
-get_kata_arch() {
-	go get -u github.com/kata-containers/tests || true
-	[ -f "${kata_arch_sh}" ] || die "Not found $kata_arch_sh"
+clone_tests_repo() {
+	# KATA_CI_NO_NETWORK is (has to be) ignored if there is
+	# no existing clone.
+	if [ -d "${tests_repo_dir}" ] && [ -n "${KATA_CI_NO_NETWORK:-}" ]; then
+	return
```
Comment (grahamwhaley) [inline scripts/lib.sh]: `return` needs an indent?

Comment (chavafg) [reply scripts/lib.sh]: ups, thanks, fixed.

**Pull request merged**